src/app/components/progress-indicator/index.html

```diff
@@ -72,22 +72,22 @@
     <sky-demo-page-property
       propertyName="title"
     >
-      Specifies the title to display with the progress item.
+      Specifies the title to display with the progress item. This property accepts <stache-code>string</stache-code> values.
     </sky-demo-page-property>
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="Progress indicator reset button properties">
     <sky-demo-page-property
-      propertyName="disabled"
-      isOptional="true"
-      defaultValue="false"
+      propertyName="progressIndicator"
     >
-      Indicates whether to disable the reset button.
+      Specifies the progress indicator component to associate with the reset button. This property accepts an array or Observable array of <stache-code>array</stache-code> of string values.
```

Comment:
I think that in the last sentence, `string` should be a code snippet and `array` should be regular text.

Reply:
Good eye, thanks! Got all jumbled. Corrected now